scripts/reloader.bash

```diff
@@ -20,9 +30,10 @@ fi
 
 
 if [ ! -z "${2}" ] && [[ "${2}" =~ ^(aliases|completion|plugins)$ ]] && [ -d "${2}/enabled" ]; then
-  # TODO: We should warn users they're using legacy enabling
+  _log_warning "Using legacy enabling, please update your bash-it version and migrate"
```

Comment:
Interesting side effect: I got this log warning three times in my local test, since I still had the empty `plugins/enabled` directories sitting around, no links in there. The migration tool does not seem to remove the empty directories.

Not sure we want to show a message in this case, as it does not hurt to have the directories. Or we update the migration tool to remove the empty directories after the migration.

Reply:
I think the better approach is to remove the directories, better open an issue about it

Reply:
Added #1631 for this.